modules/test/playwright/tests/object-web/objectDefinitions.spec.ts

```diff
@@ -88,4 +88,87 @@ test.describe('Manage object definitions through Model Builder', () => {
 			objectDefinition.id
 		);
 	});
+
+	test('linked object definitions are created when object definitions are related and put into different folders', async ({
+		apiHelpers,
+		modelBuilderPage,
+	}) => {
+		const objectFolder =
+			await apiHelpers.objectAdmin.postRandomObjectFolder();
+
+		const objectDefinition1 =
+			await apiHelpers.objectAdmin.postRandomObjectDefinition(
+				objectFolder.externalReferenceCode
+			);
+
+		const objectDefinition2 =
+			await apiHelpers.objectAdmin.postRandomObjectDefinition('default');
+
+		const objectRelationshipLabel =
+			'objectRelationshipLabel' + getRandomInt();
+		const objectRelationshipName =
+			'objectRelationshipName' + Math.floor(Math.random() * 99);
+
+		const objectRelationshipData: Partial<ObjectRelationship> = {
+			label: {
+				en_US: objectRelationshipLabel,
+			},
+			name: objectRelationshipName,
+			objectDefinitionExternalReferenceCode1:
+				objectDefinition1.externalReferenceCode,
+			objectDefinitionExternalReferenceCode2:
+				objectDefinition2.externalReferenceCode,
+			objectDefinitionId1: objectDefinition1.id,
+			objectDefinitionId2: objectDefinition2.id,
+			objectDefinitionName2: objectDefinition2.name,
+			type: 'oneToMany' as ObjectRelationshipType,
+		};
+
+		await apiHelpers.objectAdmin.postObjectRelationship(
+			objectRelationshipData
+		);
+
+		await modelBuilderPage.goto({objectFolderName: 'Default'});
+
+		await expect(
+			modelBuilderPage.getLinkedObjectDefinitionIcon(
+				objectDefinition1.name
```

Comment:
i agree with you, i was following the pattern from other tests, but i think is better to use the label and the name where they are truly needed and not because they have the same string